contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -528,17 +529,41 @@
           position = this.insertPosition(section);
         }
 
+        // All sources should be from the same region
+        const sourceRegion = drop.sources[0].region;
         const payload = {
-          id__in: data.sources.map(s => s.metadata.id),
           target: identity.metadata.id,
           position,
         };
 
-        // All sources should be from the same region
-        const sourceRegion = drop.sources[0].region;
-        return sourceRegion && sourceRegion.id === DraggableRegions.CLIPBOARD
-          ? this.copyContentNodes(payload)
-          : this.moveContentNodes(payload);
+        // When the source region is the clipboard, we want to make sure we use
+        // `excluded_descendants` by accessing the copy trees through the clipboard node ID
+        if (sourceRegion && sourceRegion.id === DraggableRegions.CLIPBOARD) {
+          return Promise.all(
+            data.sources.map(source => {
+              const trees = this.getCopyTrees(source.metadata.clipboardNodeId, true);
+
+              if (trees.length === 0) {
+                return Promise.resolve();
+              } else if (trees.length > 1) {
+                throw new Error(
+                  'Multiple copy trees are unexpected for drag and drop copy operation'
+                );
+              }
+
+              return this.copyContentNode({
+                ...payload,
+                id: source.metadata.id,
+                excluded_descendants: get(trees, [0, 'extra_fields', 'excluded_descendants'], {}),
+              });
+            })
+          );
+        }
```

Comment:
Uses `clipboardNodeId` and `getCopyTrees` to ensure that what we end up copying matches what they would expect after seeing it on the clipboard. Specifically, the clipboard makes use of an `excluded_descendants` parameter which we should pass along when copying. 

For instance, if the user duplicated a topic already on the clipboard, with a portion of its descendant resources unselected, then when we attempt to drag-n-drop that topic, we want to be sure and not copy all the descendants.

Reply:
Thanks for the helpful example